juriscraper/OpinionSiteAspx.py

```diff
@@ -0,0 +1,74 @@
+from lxml import html
+from juriscraper.OpinionSite import OpinionSite
+
+
+class OpinionSiteAspx(OpinionSite):
+    def __init__(self, *args, **kwargs):
+        super(OpinionSiteAspx, self).__init__(*args, **kwargs)
+        self.spoof_user_agent = False
+
+    def _get_soup(self, url):
+        """Download a page of the site. Can be called multiple times.
+
+        Either the first page of the site with GET, if self.data is empty, or
+        a subsequent page with POST, if it is filled.
+        """
+        if self.spoof_user_agent:
+            self.request["headers"] = {
+                "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.122 Safari/537.36",
+            }
+
+        if not self.data:
+            r = self.request["session"].get(url)
+        else:
+            r = self.request["session"].post(url, data=self.data)
+        self.soup = html.fromstring(r.text)
+
+    def _get_data_template(self):
+        """Returns a template for data that should be POSTed to an ASPX page.
+
+        This can include any number of key/values that are the same between
+        pages. It can also contain the special keys __VIEWSTATE and __EVENTTARGET
+        whose values are ignored and set to the appropriate value in the
+        _update_data method.
```

Comment:
I'm not sure how much I care about this in Juriscraper, but we are moving towards more formal docstrings in general. The usual format is:

```
"""One line summary

Longer paragraph explaining stuff

:param self: What self is
:type self: What type self is
:return: What it returns
:rtype: The type of the returned object
"""
```

PyCharm has actually started issuing warnings on these type hints. It's neat.

Reply:
if only we were on p3 and could use proper typing :)

Reply:
I'm not sure exactly what you're asking for. I'm already following this format, except that I didn't document the parameter `url`. I've fixed that now, is it fine?

Reply:
Yep!